### DISCUSSION TITLE
Not able to find a source or destination in Airbyte Cloud

### DISCUSSION THREAD

**Discussion by octavia-faq** [airbytehq/airbyte#32248] [Connector Questions] [answered]
When navigating the Airbyte catalog documentation, you'll encounter numerous sources and destinations. I have located the connector documentation required for my project. I am currently utilizing Airbyte Cloud, and when attempting to create a new source or destination, I cannot find the connector in the list. What might be the reason for this issue?

**Comment by marcosmarxm** [comment, marked as answer]
You can check the Airbyte catalog [here](https://docs.airbyte.com/integrations/).This page will display what connectors are available in Airbyte OSS and Airbyte Cloud. Some connectors aren't available in Airbyte Cloud because they don't have the minimal tests to ensure their quality.
